<launch>
<!-- Launch file for INNO LiDAR -->

	<arg name="rviz" default="true" />
	<node pkg="rosbag" type="play" name="rosbag_play" args="--wait-for-subscribers -r 1 $(find inno_ligo)/data/test.bag"/>
	<!-- <node pkg="rosbag" type="play" name="rosbag_play" args="-d 10 -r 1 $(find inno_lio)/data/test.bag"/> -->

	<rosparam command="load" file="$(find inno_ligo)/config/inno.yaml" />
    <param name="max_iteration" type="int" value="5" />

    <!-- <node pkg="inno_ligo" type="ligo_test" name="ligo_test" output="screen" launch-prefix="gdb -ex run -args" />  -->
    <node pkg="inno_ligo" type="ligo_test" name="ligo_test" output="screen" /> 

	<group if="$(arg rviz)">
	<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find inno_ligo)/config/inno_lio.rviz" />
	</group>

</launch>
